packages/console/src/pages/SignInExperience/components/SignInMethodsForm.tsx

```diff
@@ -18,6 +18,7 @@ const signInMethods = Object.values(SignInMethodKey);
 const SignInMethodsForm = () => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { register, watch, control, setValue } = useFormContext<SignInExperienceForm>();
+  const oldPrimaryMethod = useRef<SignInMethodKey>();
```

Comment:
I think we can use remote data as the "old method", don't need to remember the "unsaved" value.

Reply:
Updated, PTAL the new logic.